addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -158,6 +158,11 @@ registerModel({
             }
             this.threadView.topbar.openInvitePopoverView();
         },
+        open() {
+            if (!this.discussView) {
```

Comment:
guard is not necessary, insert will re-use the existing one as it is the same identifying field